bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/guice/BobcumberModule.java

```diff
@@ -20,26 +20,37 @@
 package com.cognifide.qa.bb.cumber.guice;
 
 import com.cognifide.qa.bb.assertions.soft.SoftAssertionsAspect;
+import com.cognifide.qa.bb.loadable.CucumberLoadableProcessorFilter;
+import com.cognifide.qa.bb.loadable.LoadableProcessorFilter;
 import com.google.inject.AbstractModule;
 import com.google.inject.matcher.Matchers;
 
+import com.google.inject.multibindings.Multibinder;
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.But;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 
+import static com.google.inject.matcher.Matchers.any;
+
 public class BobcumberModule extends AbstractModule {
 
   @Override
   protected void configure() {
+
+    bindListener(any(), new CucumberLoadableProcessorFilter());
     // soft assertion aspect
     SoftAssertionsAspect softAssertAspect = new SoftAssertionsAspect();
     requestInjection(softAssertAspect);
-    bindInterceptor(Matchers.any(), Matchers.annotatedWith(When.class), softAssertAspect);
```

Comment:
I think it is better to avoid such changes in pull request.
Formatting changes with new code at the same time.
Could we leave `Matchers.any()` for now to have simpler pull request?
(Should have)